htdocs/adminhistory.php

```diff
@@ -33,7 +32,7 @@
     $error = $translate->t('Cache not found', '', '', 0);
 } elseif ($cache_id > 0) {
     $showhistory = true;
-    $cache = new Cache($cache_id);
+    $cache = new cache($cache_id);
     $cache->setTplHistoryData(0);
```

Comment:
To correct myself: The bug was not introduce during refactoring, but was always there. It never was noticed because PHP class names are case-insensitve, but now appeared because composer class loading is case sensitive.